### PR TITLE
Fix bug with redcap logging

### PR DESCRIPTION
**Pull request by ao508**
Signed-off-by: Angelica Ochoa <ochoaa@mskcc.org>

### CODE REVIEW

File: cvr/pom.xml [hunk 1]
```diff
@@ -14,12 +14,6 @@
   </parent>
 
   <dependencies>
-    <dependency>
```
Comment (ao508) [inline cvr/pom.xml:17]: this is already in the master pom so I moved it out of here

File: cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/BatchConfiguration.java [hunk 2]
```diff
@@ -772,4 +784,76 @@ public JobExecutionDecider svFusionsStepExecutionDecider() {
     public JobExecutionDecider segStepExecutionDecider() {
         return decideStepExecutionByDatatypeForStudyId("seg");
     }
+
+    // general spring batch configuration
+    @Value("org/springframework/batch/core/schema-drop-sqlite.sql")
+    private Resource dropRepositoryTables;
+
+    @Value("org/springframework/batch/core/schema-sqlite.sql")
+    private Resource dataRepositorySchema;
```
Comment (ao508) [inline cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/BatchConfiguration.java:793]: all of the changes in the BatchConfiguration classes are the same. 

These are required beans that need to be explicitly defined in the newer version of Spring Batch that we're using now

File: gene/src/main/java/org/mskcc/cmo/ks/GeneDataPipeline.java [hunk 3]
```diff
@@ -49,14 +49,14 @@
 
 @SpringBootApplication
 public class GeneDataPipeline {
-    
-    private static final Log LOG = LogFactory.getLog(GeneDataPipeline.class);
-    
+
+    private final static Logger LOG = Logger.getLogger(GeneDataPipeline.class);
```
Comment (ao508) [inline gene/src/main/java/org/mskcc/cmo/ks/GeneDataPipeline.java:53]: standardizing the logger used

File: cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/BatchConfiguration.java (hunk 2 above)
Comment (sheridancbio) [reply cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/BatchConfiguration.java:793]: ✅

**Review by sheridancbio — APPROVED** [review]
these logger libraries continue to haunt us ..
nice work

**Pull request merged**